src/renderer/prefComponents/general/index.vue

```diff
@@ -2,12 +2,12 @@
   <div class="pref-general">
     <h4>General</h4>
     <bool
-      description="Automatically save the content being edited."
+      description="Automatically save changes."
```

Comment:
I think >Automatically save document changes< would be easier to understand, what do you think @Rexogamer?

Reply:
I agree, and I've changed it in the latest commit 👍